standardwebpage.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset="UTF-8">
+    <script src="https://kit.fontawesome.com/24a476b79d.js" crossorigin="anonymous"></script>
+    <style>
+      body{
+        background-color: #bbecff;
+      }
+      ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+        background-color: rgb(10, 17, 119);
+        font-size: larger;
+      }
+      li {
+        float: right;
+      }
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.About{
+  max-width: 1000px;
+  margin: auto;
+  padding: 20px;
+  text-align: justify;
+}
+h1{
+  font-size: 80px;
+}
+
+p{
+            font-family:"Poppins";
+        }
+        .card {
+            box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+            width: 250px;
+            height: 450px;
+            margin:auto;
+            text-align: center;
+            color: white;
+            background-color: white;
+            margin-top: 40px;
+            display: inline-block;
+            margin-left: 100px;
+        }
+        .title {
+            color:rgb(10, 75, 196);
+            font-size: 18px;
+        }
+        h2{
+            font-family: "Poppins",sans-serif;
+            color:rgb(0, 0, 0);
+        }
+
+a {
+  font-size: 25px;
+  color: black;
+  padding:10px;
+}
+    </style>
+</head>
+<body>
+  <div class = "navbar">
+    <ul>
+      <li><a href="https://memoriestejastapas.netlify.app/">Sign In/ Sign Up</a></li>
+      <li><a href="#news">Contact Us</a></li>
+      <li><a href="#home">About Us</a></li>
+      <li style="float:left"><a class="active" href="https://memoriestejastapas.netlify.app/">Athavani</a></li>
+    </ul>
+  </div>
+  <div class = "image1">
+    <img src = "C:\Users\Kanishka\Downloads\Memories are forever (3).png" height="700px" width="1500px">
```

Comment:
You have used the *absolute* path for the image, which was stored in your local Machine. I will suggest adding this image to the project folder and then use the *relative* path.